dashboard/test/models/script_test.rb

```diff
@@ -70,59 +69,6 @@ def populate_cache_and_disconnect_db
     ActiveRecord::Base.connection.stubs(:execute).raises 'Database disconnected'
   end
 
-  test 'login required setting in script file' do
```

Comment:
Can you point me to the equivalent test for this? Or do you think its just not necessary?

Reply:
good question, I guess there is not really an equivalent of this. the sample scripts such as https://github.com/code-dot-org/code-dot-org/blob/0a8563811853d4871b7ba7d211bc92eec546ba6e/dashboard/test/fixtures/test-serialize-seeding-json.script_json#L5 set it to false.

My short answer is I think it is not necessary. The logic to add a new field for serialization only requires that you add it to the serializer, here: https://github.com/code-dot-org/code-dot-org/blob/61e0328e487582db37fb07453f629ef03496397b/dashboard/lib/services/script_seed.rb#L691

For properties (as opposed to fields / db columns), it isn't even necessary to add anything to a serializer.

The old script seed tests had tests which comprehensively verified that every field and property could be set via test-all-properties.script and test-all-fields.script. this was because you had to write boilerplate logic for each field in order to make it work.

Does that make sense, or do you think more test coverage is in order here?

Reply:
That makes sense. That was my leaning as well. Thanks!